orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestEvaluator.java

```diff
@@ -66,17 +67,20 @@
   private final ContextParameterProcessor contextParameterProcessor;
   private final OortService oortService;
   private final RetrySupport retrySupport;
+  private final Boolean failOnUnknownKeys;
 
   @Autowired
   public ManifestEvaluator(
       ArtifactUtils artifactUtils,
       ContextParameterProcessor contextParameterProcessor,
       OortService oortService,
-      RetrySupport retrySupport) {
+      RetrySupport retrySupport,
+      @Value("${manifest.evaluator.failOnUnknownKeys:true}") Boolean failOnUnknownKeys) {
```

Comment:
Two things:
1. As of Spring 2 you need to use the snake case otherwise the props won't load
```suggestion
      @Value("${manifest.evaluator.fail-on-unknown-keys:true}") Boolean failOnUnknownKeys) {
```

2. (you don't need to change this, just for future reference) we prefer the use of `ConfigurationProperties` pattern over `@Value` annotations because a) they don't suffer from these casing issues and b) they are much more easily discoverable and docs for the can be generated automatically